### DISCUSSION TITLE
Predictive back on all Androids

### DISCUSSION THREAD

**Discussion by nevrozza** [arkivanov/Decompose#719] [Q&A] [answered]
Is there way to use predictive back on all androids (like ios, cuz it hasn't predictive back)?

**Comment by arkivanov** [comment, marked as answer]
Unfortunately, there is no reliable way of implementing it on all Android versions. Starting with Android 10 there is [Gesture Navigation](https://developer.android.com/develop/ui/views/touch-and-input/gestures/gesturenav), which always intercepts gestures close to the screen edges.
